crates/revm/src/evm_impl.rs

```diff
@@ -192,9 +191,9 @@ impl<'a, GSPEC: Spec, DB: Database, const INSPECT: bool> EVMImpl<'a, GSPEC, DB,
         &mut self,
         caller: H160,
         gas: &Gas,
-    ) -> Result<(Map<H160, Account>, Vec<Log>), Return> {
+    ) -> (Map<H160, Account>, Vec<Log>, u64) {
```

Comment:
The function signature change should be OK given that this function is not exposed